micromanager_gui/main_window.py

```diff
@@ -127,6 +121,27 @@ def __init__(self, viewer: napari.viewer.Viewer, remote=False):
         self.viewer.layers.selection.events.active.connect(self.update_max_min)
         self.viewer.dims.events.current_step.connect(self.update_max_min)
 
+        @self.viewer.mouse_drag_callbacks.append
+        def get_event(viewer, event):
```

Comment:
can we put this as a method instead of decorated in the init? 

```
self.viewer.mouse_drag_callbacks.append(self._get_event)
```

Reply:
sure thing